core/processor/src/main/java/io/quarkus/annotation/processor/generate_doc/ConfigDocItemScanner.java

```diff
@@ -61,30 +61,32 @@ public void addConfigGroups(TypeElement configGroup) {
      */
     public void addConfigRoot(final PackageElement pkg, TypeElement clazz) {
         if (pkg.toString().startsWith(IO_QUARKUS_TEST_EXTENSION_PACKAGE)) {
-            return;
+            //return;
         }
```

Comment:
What's the reasoning behind this change?

Reply:
Ops. Shouldn't be there. Was using these to test the generation with the test extension.